main.py

```diff
@@ -238,9 +238,9 @@ def upgrade(old_version, new_version):
 
 
 if __name__ == '__main__':
-    upgrade('2.6.0', '2.6.9')
-    upgrade('2.6.1', '2.6.9')
-    upgrade('2.6.5', '2.6.9')
-    upgrade('2.6.6', '2.6.9')
-    upgrade('2.7.2', '2.7.5')
-    upgrade('2.7.3', '2.7.5')
+    upgrade('2.6.0', '2.7.6')
+    upgrade('2.6.1', '2.7.6')
+    upgrade('2.6.5', '2.7.6')
+    upgrade('2.6.6', '2.7.6')
+    upgrade('2.7.2', '2.7.6')
+    upgrade('2.7.3', '2.7.6')
```

Comment:
We're missing 3.0.3 ([used by a few apps](https://docs.google.com/spreadsheets/d/1Lt-uarJm1xaCEC58KdSixsa5LsBSmNXym37SbuXbrcI/edit?pli=1#gid=1480786499)) to 3.0.4.

Reply:
Thanks, I've added it now.